src/main/java/Updater/Updater.java

```diff
@@ -0,0 +1,43 @@
+package Updater;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.awt.Desktop;
+
+public class Updater {
+
+    public static void main(String[] args) {
+        String oldFilePath = "";
+        String newFilePath = "";
+        replaceUpdate(oldFilePath , newFilePath);
+    }
+
+    public static void replaceUpdate(String oldFilePath ,String newFilePath){
```

Comment:
@kmr-rohit The whole code present in `replaceUpdate` method should be in the `main()` method. There should not be any more methods in this class.

Reply:
Changed As requested

Reply:
Okay :+1: